lms/djangoapps/discussion/rest_api/views.py

```diff
@@ -568,11 +564,22 @@ class LearnerThreadView(APIView):
 
     **GET Thread List Parameters**:
 
+        * user_id: (Required) (Integer) ID of the user whose active threads are required
+
         * page: The (1-indexed) page to retrieve (default is 1)
 
         * page_size: The number of items per page (default is 10)
     """
 
+    authentication_classes = (
+        JwtAuthentication,
+        BearerAuthentication,
+        SessionAuthentication,
+    )
+    permission_classes = (
+        permissions.IsAuthenticated,
```

Comment:
Can any authenticated user have access to this API, or is it for staff/admin only?